lib/js_of_ocaml/dom_html.mli

```diff
@@ -2201,9 +2204,13 @@ class type window = object
 
   method scrollBy : number_t -> number_t -> unit meth
 
-  method sessionStorage : storage t readonly_prop
+  (* These two properties are not available on non-Web environments
+     (for instance, Web workers, node). So we keep the [optdef]
+     annotation for now. *)
```

Comment:
Isn't this true for most method inside `window` ?

Reply:
Yes, indeed. The point is that existing programs have been written (and tested) with this in mind but, if one is not careful, one might introduce a bug when removing the test when accessing the local/session storage.